project_proposal/stakeholders.md

```diff
@@ -0,0 +1,9 @@
+### Stakeholders
+
```

Comment:
General comment: 
Avoid using abbreviations such as "don't", "there's" etc. to make it more formal.

Reply:
I expanded all cases where I had shortened the words